app/modules/intelligence/agents/hatchet_local_bootstrap.py

```
@@ -0,0 +1,204 @@
+    Path(env_path).write_text(content)
+
+
+def ensure_hatchet_database(
+    run: Runner = _run,
+    *,
+    container: str = DEFAULT_CONTAINER,
+    user: str = DEFAULT_DB_USER,
+    db_name: str = DEFAULT_DB_NAME,
+) -> bool:
+    """Create the ``hatchet`` database if absent. Returns True if it was created."""
+    select_sql = f"SELECT 1 FROM pg_database WHERE datname='{db_name}'"
```

Ruff (0.15.14)

[error] Possible SQL injection vector through string-based query construction

(S608)